schematic_api/api/__main__.py

```diff
@@ -13,7 +13,7 @@ def main():
 
     # Launch app
     # CORS(app, resources={r"*": {"origins": "*"}})
-    app.run(host=host, port=port, debug=False)
+    app.run(host=host, port=port, debug=True)
```

Comment:
this will also be reverted back to "false" before merging

Reply:
Nit: Add TODO, I feel like there might be a way to start the app with `--debug` flag that does `debug=debug`.  The reason I say TODO is because you don't want to change too many things in a PR, just try to stick with the original purpose of the PR.

Reply:
@thomasyu888 Thanks for the advice. I will add a comment to the code.